### ISSUE TITLE
[BUG]: Searching for TV shows doesn't find playlist results, but the web version actually exists

### ISSUE THREAD

**Issue by ChenYuZe519** [yuliskov/SmartTube#4106]
### Checklist

- [X] I made sure that there are *no existing issues* - [open](https://github.com/yuliskov/SmartTube/issues) or [closed](https://github.com/yuliskov/SmartTube/issues?q=is%3Aissue+is%3Aclosed) - which I could contribute my information to.
- [X] I have read the [FAQ](https://github.com/yuliskov/SmartTube#faq) and my problem isn't listed.
- [X] I have taken the time to fill in all the required details. I understand that the bug report will be dismissed otherwise.
- [X] This issue contains only one bug.

### Affected version

beta

### Device Type

Smart TV/Box

### Affected Android

7

### Steps to reproduce the bug

Search

### Actual behavior

Results for Playlists not found

**Comment by not-a-coder-at-all** [comment]
Yes, the issue started from 24.53 Stable, but 24.37 Stable didn't have this issue.
